draft-ietf-rtcweb-jsep.xml

```diff
@@ -3732,11 +3736,18 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                 MUST be returned.
                 <list style="symbols">
 
-                    <t>If the description is of type "offer", and the
-                    ICE ufrag or password changed from the previous
-                    remote description, as described in Section 9.1.1.1
-                    of <xref target="RFC5245" />, mark that an ICE
-                    restart is needed.</t>
+                    <t>If the ICE ufrag or password changed from the previous
+                    remote description, then an ICE restart is needed,
+                    as described in Section 9.1.1.1
+                    of <xref target="RFC5245" />
+                    If the description is of type "offer", mark that an ICE
+                    restart is needed. If the description is of type
+                    "answer" and the current local description is also
+                    an ICE restart, then signal the ICE agent to begin
+                    checks as described in Section 9.3.1.1 of <xref target="RFC5245" />.
+                    It is an error for an answer to change the ufrag
```

Comment:
Agreed. I assume it is also an error for the answer to not change the ufrag and password if the offer did. 

Also, should this use MUST etc instead of "It is an error"?